### DISCUSSION TITLE
Android Modal component in bridgeless does not get BridgelessReactContext

### DISCUSSION THREAD

**Discussion by douglowder** [reactwg/react-native-new-architecture#186] [Q&A] [answered]
On Android bridgeless, I would like to know why the ReactContext of views within the root view are of class BridgelessReactContext, but the context passed to a Modal view is still ThemedReactContext (as it is for non-Fabric or non-bridgeless).

I'm hitting an issue where the Modal view needs to get access to a native or JS module to send events, but this is not working in bridgeless because there is no React instance or Catalyst instance, as the ThemedReactContext expects when accessing a module.

**Comment by cortinico** [comment, marked as answer]
> On Android bridgeless, I would like to know why the ReactContext of views within the root view are of class BridgelessReactContext, but the context passed to a Modal view is still ThemedReactContext (as it is for non-Fabric or non-bridgeless).

This is a known issue. @RSNara and @fabriziocucci have been working on fixing it and can tell you more about it

**Reply by douglowder** [reply]
Thanks @cortinico !!
